src/UI/Implementation/Component/Input/Field/Renderer.php

```diff
@@ -707,9 +705,14 @@ protected function renderDurationField(F\Duration $component, RendererInterface
                                                                     'useCurrent' => false
         ]);
         $input_html .= $default_renderer->render($input);
+
+        $start = strpos($input_html, '<input id="') + strlen('<input id="');
+        $stop = strpos($input_html, '"', $start);
+        $first_input_id = substr($input_html, $start, $stop - $start);
+
```

Comment:
don't shoot me for this (now), the PR is primarily about the moving of ids.
We'll come up with a better way, here